lib/share_progress/button.rb

```diff
@@ -21,7 +35,8 @@ def destroy(id)
 
       def all(options={})
```

Comment:
Neal, just another point is that with keyword arguments you're providing documentation through code. 
```ruby
def all(offset: 5, limit: 100)
```

To someone reading your code, it's immediately apparent what the API requires.

Also, it's better we set defaults our side for predictable behaviour. What do you think?